### PR TITLE
Update deploy-manage-removable-storage-group-policy.md

### LINKED ISSUE #13113: USB control is a MDE-P1 feature
USB Control is a MDE-P1 feature , which can be bought as a standalone. There is no explicit reliance on ME3

The request would be to update the text to mention that for endpoint USB control, the license required is at minimum MDE-P1. Higher band licenses of M365 E3 which includes MDE-P1, or M365 E5, which includes MDE-P1 plus MDE-P2 , can of course also suffice)

===
This page says both MDE-P1 and MDE-P2 support the USB control
https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control?view=o365-worldwide

This MDE-Plan1 links says its included
https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/defender-endpoint-plan-1?view=o365-worldwide
Device control : Sometimes threats to your organization's devices come in the form of files on removable drives, such as USB drives. Defender for Endpoint includes capabilities to help prevent threats from unauthorized peripherals from compromising your devices. You can configure Defender for Endpoint to block or allow removable devices and files on removable devices.
To learn more, see [Control USB devices and removable media](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/control-usb-devices-using-intune?view=o365-worldwide).

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 48609e18-20d4-ff9c-fbb2-d837a8d47adb
* Version Independent ID: 48609e18-20d4-ff9c-fbb2-d837a8d47adb
* Content: [Deploy and manage Removable Storage Access Control using group policy](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/deploy-manage-removable-storage-group-policy?view=o365-worldwide#licensing-requirements)
* Content Source: [microsoft-365/security/defender-endpoint/deploy-manage-removable-storage-group-policy.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/deploy-manage-removable-storage-group-policy.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
added more information to the license requirements

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/13113

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/deploy-manage-removable-storage-group-policy.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ You can use the following properties to create a removable storage group.
 
 ## Licensing requirements
 
-Before you get started with Removable Storage Access Control, you must confirm your [Microsoft 365 subscription](https://www.microsoft.com/microsoft-365/compare-microsoft-365-enterprise-plans?rtc=2). To access and use Removable Storage Access Control through group policy, you must have Microsoft 365 E3 or E5.
+Before you get started with Removable Storage Access Control, you must confirm your [Microsoft 365 subscription](https://www.microsoft.com/microsoft-365/compare-microsoft-365-enterprise-plans?rtc=2). To access and use Removable Storage Access Control through group policy, you must have a standalone Microsoft defender for endpoint Plan 1 or Microsoft 365 E3 which includes Microsoft defender for endpoint Plan 1 or Microsoft 365 E5 which includes Microsoft defender for endpoint Plan 1 and 2.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/deploy-manage-removable-storage-group-policy.md]:
```suggestion
Before you get started with Removable Storage Access Control, you must confirm your [Microsoft 365 subscription](https://www.microsoft.com/microsoft-365/compare-microsoft-365-enterprise-plans?rtc=2). To access and use Removable Storage Access Control through group policy, you must have a standalone Microsoft Defender for Endpoint Plan 1 or Microsoft 365 E3 which includes Microsoft Defender for Endpoint Plan 1 or Microsoft 365 E5 which includes Microsoft Defender for Endpoint Plan 1 and 2.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**